hack/update-codegen.sh

```diff
@@ -6,8 +6,7 @@ set -o pipefail
 
 SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
 CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ${GOPATH}/src/k8s.io/code-generator)}
-
-vendor/k8s.io/code-generator/generate-groups.sh "deepcopy,client,informer,lister" \
+bash $CODEGEN_PKG/generate-groups.sh "deepcopy,client,informer,lister" \
   volcano.sh/apis/pkg/client volcano.sh/apis/pkg/apis \
```

Comment:
which branch of apis do we use?

Reply:
l used master branch in this PR

Reply:
it's better to make it configurable; if someone, checkout this script in different branch, it'll also use master to generate client-go :)

Reply:
l don't really catch your mean. If user checkout to other branch and use this script, the client will be generated according to the apis in that branch, while not master